### DISCUSSION TITLE
Issue with Certificate Request Error: Certificate provisioning will retry automatically in a short period, please be patient.

### DISCUSSION THREAD

**Discussion by spaziog** [community/community#101892] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've got an issue where the certificate doesn't work on my custom profile.

I've got this error in the setting page "Certificate Request Error: Certificate provisioning will retry automatically in a short period, please be patient."

I've already verified again the domain and set the CAA to letsencript value.

Below my DNS settings, note that my provider add some settings by default and I can't remove them.
|name                                             |type |value                                 |ttl |
|-------------------------------------------------|-----|---------------------------------------|----|
|www.spaziogege.it.                               |CNAME|spaziogege.it.                         |300 |
|_caldav._tcp.spaziogege.it.                      |SRV  |0 0 2079 cpanel.spaziogege.it.         |300 |
|_caldav._tcp.spaziogege.it.                      |TXT  |"path=/"                               |300 |
|_carddav._tcp.spaziogege.it.                     |SRV  |0 0 2079 cpanel.spaziogege.it.         |300 |
|_carddav._tcp.spaziogege.it.                     |TXT  |"path=/"                               |300 |
|_caldavs._tcp.spaziogege.it.                     |SRV  |0 0 2080 cpanel.spaziogege.it.         |300 |
|_caldavs._tcp.spaziogege.it.                     |TXT  |"path=/"                               |300 |
|_carddavs._tcp.spaziogege.it.                    |SRV  |0 0 2080 cpanel.spaziogege.it.         |300 |
|_carddavs._tcp.spaziogege.it.                    |TXT  |"path=/"                               |300 |
|_autodiscover._tcp.spaziogege.it.                |SRV  |0 0 443 panelemaildiscovery.cpanel.net.|300 |
|_github-pages-challenge-spaziogege.spaziogege.it.|TXT  |"REDACTED"       |300 |
|spaziogege.it.                                   |CAA  |0 issue "letsencrypt.org"              |300 |
|spaziogege.it.                                   |TXT  |"v=spf1 +a +mx -all"                   |300 |
|spaziogege.it.                                   |NS   |dns4.netsons.net.                      |3600|
|spaziogege.it.                                   |NS   |dns3.netsons.net.                      |3600|
|spaziogege.it.                                   |NS   |dns2.netsons.net.                      |3600|
|spaziogege.it.                                   |NS   |dns1.netsons.net.                      |3600|
|spaziogege.it.                                   |A    |185.199.111.153                        |300 |
|spaziogege.it.                                   |A    |185.199.110.153                        |300 |
|spaziogege.it.                                   |A    |185.199.109.153                        |300 |
|spaziogege.it.                                   |A    |185.199.108.153                        |300 |

**Comment by yoannchaudet** [comment]
Your `CNAME` should point at `spaziogege.github.io` (as per our [doc](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site?platform=mac#configuring-a-subdomain)). Technically your configuration is correct but we have validation in place today that prefers what I am suggesting. I'll see if we can improve that in the future.

**Reply by spaziog** [reply]
many thanks @yoannchaudet , it worked!
